examples/js/loaders/FBXLoader2.js

```diff
@@ -571,11 +571,10 @@
 
 							var faceVertexBuffer = [];
 							var polygonIndex = 0;
-							for ( var polygonVertexIndex = 0; polygonVertexIndex < indexBuffer.length; ++ polygonVertexIndex ) {
+							indexBuffer.forEach( function ( vertexIndex, polygonVertexIndex, indexBuffer ) {
```

Comment:
Coming from a C++ background, this completely baffles me.  But performance is performance, so I'll go ahead and remove the for-each's